### ISSUE TITLE
Add build script

### ISSUE THREAD

**Issue by davelab6** [TypeNetwork/fb-Decovar#23]
https://github.com/TypeNetwork/fb-Decovar/blob/master/sources/2-build/Decovar.designspace
 exists but doesn't appear referenced in any build script (https://github.com/TypeNetwork/fb-Decovar/search?utf8=%E2%9C%93&q=Decovar.designspace&type=)

(Whereas in Amstelvar it is in
https://github.com/TypeNetwork/fb-Amstelvar/blob/a879f3fbcc93826609e0d2bda6a90f0df0748cce/scripts/build.sh#L1)

**Comment by djrrb** [comment]
I’ve added one similar to Amstelvar. https://github.com/TypeNetwork/fb-Decovar/pull/24

Note that this does not cover the manual addition of a STAT table. Hopefully this will be added to fonttools varLib soon, so maybe not worth building in yet?

**Comment by davelab6** [comment]
I guess just add a comment to the build script to show where the manual
step occurs.... although AFAIK its like... you TTX the binary, copy the
stat table into the directory with the right filename, patch the index TTX
file, and re-run ttx on it to zip the XML back into the binary... so that
seems scriptable

**Comment by djrrb** [comment]
totally scriptable...do we need parity between amstelvar and decovar build processes? I would do this in python instead of a shell script...

**Comment by davelab6** [comment]
I don't need parity, I want a random type designer to be able to take the source files and reproduce the published binary with no fuss :) But that is rather ideal, and all that the contract requires is that you provide what you actually used to make your binary yourself, and adequate documentation for someone to reproduce what you did. 

Section 15.2.2 provides more context if you want to nerd out on this stuff :) https://copyleft.org/guide/comprehensive-gpl-guidech16.html

**Comment by sberlow** [comment]
So a python script is ok?

**Comment by davelab6** [comment]
> So a python script is ok?

Absolutely! waf and SCons are the popular Python build systems, or you call roll your own in "make file style" (as in https://github.com/sphinx-doc/sphinx/issues/3196), or just have a simple imperative python script file...

**Comment by djrrb** [comment]
i'll make something basic tomorrow and we can always add a build system around it...not very knowledgeable about these different approaches

**Comment by davelab6** [comment]
Thank you!! (Simple is good :)

**Comment by djrrb** [comment]
Well, the good news is that the latest fontTools now writes its own STAT table, so we no longer need to include a more complex build script to address it. :-)

The bad news is that the build script is finding some incompatibilities and not generating the VF as expected. So just wanted to let you know I’m still on top of automating the build process, but it might take a bit longer than expected, and the changes will affect the sources and not only the script itself.

**Comment by davelab6** [comment]
Ok cool, I appreciate you looking into it

**Comment by dberlow** [comment]
Change the sources?

**Comment by djrrb** [comment]
no design changes...just to make them compatible or whatever needs doing to get Decovar to build from source without manual intervention

**Comment by dberlow** [comment]
Who said design changes?;)

What I’m interested in are the source changes required for a style to properly build a stat table, if that is not already set up by nature and nurture in most fonts. 

Decovar will always be an alien.

**Comment by djrrb** [comment]
Haha ok good.

A basic STAT table is now being built by fontTools varLib (see commit [here](https://github.com/fonttools/fonttools/commit/8af430997d00497f9aa5b0bd71af7922f7669ff2)), so should now be included in all versions built with the latest fontTools.

There may be more source changes in store once more STAT-related data is specified in the designspace file (see discussion [here](https://github.com/LettError/designSpaceDocument/issues/8))

**Comment by djrrb** [comment]
I think this is ready to roll, see #25. Please review, merge, and close if this looks good.
